sample/src/main/res/layout/activity_base.xml

```diff
@@ -26,4 +26,13 @@
         app:newEventTimeResolutionInMinutes="15"
         app:showHalfHours="false" />
 
+    <TextView
+        android:id="@+id/draggable_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@android:color/white"
+        android:background="#ffff0000"
+        android:padding="3dp"
+        android:text="@string/drag_me"/>
```

Comment:
this view appears in sample app, not in library, i see no reason to make setter or property in this case